### ISSUE TITLE
html5 videos not working on qutebrowser

### ISSUE THREAD

**Issue by Okyougotme** [Homebrew/brew#3623]
Hi All, 
Currently html5 videos are not working on my home brew version of qutebrowser. Which complier should I use to  recompile Qt5 ? Could I possibly use GCC in Terminal to compile, Atom with run code in Atom package or do I need to download Xcode? Also I already have developer tools installed in Terminal. I’m new to the browser reason for all the questions.

![34371319-1cb3d564-ea9a-11e7-9a63-aa24bd889cf8](https://user-images.githubusercontent.com/34583010/34478927-a94b6452-ef71-11e7-8ea9-b863f591f564.jpg)

I get this error when attempting to compile using Atom: uninitialized constant Formula (NameError)
How can I resolve this issue. 

![34371674-35a4297c-ea9d-11e7-8e9c-5d23b7558ef7](https://user-images.githubusercontent.com/34583010/34478939-bfeda6f2-ef71-11e7-9c15-d11cbce10623.jpg)

Should I change the name from the source file: usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/qt.rb:3

![34372271-83a34096-eaa1-11e7-85b7-44387d64cce8](https://user-images.githubusercontent.com/34583010/34478949-d010660a-ef71-11e7-805e-c78ac73d0ea1.jpg)

Thanks in advance

**Comment by ilovezfs** [comment]
You'll need to seek help here https://github.com/qutebrowser/qutebrowser/issues

**Comment by Okyougotme** [comment]
I already attempted to do so, the developer ask me to seek assistance from home brew issues since QtWebKit is disable by default

**Comment by ilovezfs** [comment]
You can tell the developer you need help using qutebrowser with the `qt@5.5` formula, which still has QtWebKit support.

**Comment by MikeMcQuaid** [comment]
Sorry, we don't provide support for individual packages here. You'll need to figure it out yourself. In addition, from the issue template:

> **Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you do not read fill out the issue template repeatedly you will be blocked from ever submitting issues to Homebrew again.**

**Comment by Okyougotme** [comment]
I filled out the information to the best of my knowledge, I created this github account for help. Since I’m new to github I was not aware that their were checklist.  very harsh, blocking someone for not knowing something, how authoritarian of you. Go ahead and block me for seeking out help.

**Comment by MikeMcQuaid** [comment]
@Okyougotme I didn't and won't block you unless it became a pattern of behaviour. You deleted the checklist when you creating the issue, it'll be there if you ever create any in future. It's not authoritarian; we just have literally over a million users and thousands of packages and can't provide support for them all on a volunteer run project.

**Comment by The-Compiler** [comment]
qutebrowser's maintainer here - let me clear some things up and provide some context:

@Okyougotme [asked](https://github.com/qutebrowser/qutebrowser/issues/3437) in the qutebrowser repo how to make all videos play correctly. I told them they have to recompile Qt (or rather, QtWebEngine) with support for proprietary codecs. After [providing](https://github.com/qutebrowser/qutebrowser/issues/3439) some more details in a follow-up issue I got a lot of Homebrew related questions I can't answer (what compiler to use, why there are errors highlighted in their editor, etc.), so @jgkamat and I pointed them to how to get [Homebrew support](https://github.com/Homebrew/brew#community).

QtWebKit is unrelated FWIW, and I'm not sure what "since QtWebKit is disable by default" is supposed to mean. qutebrowser can use QtWebKit (but only the [updated fork](https://github.com/annulen/webkit/)), but the default is QtWebEngine, and the question is withj QtWebEngine.
